testing/src/test/java/io/stargate/it/BaseOsgiIntegrationTest.java

```diff
@@ -73,15 +78,43 @@
     unFinal();
   }
 
-  static Starter stargateStarter;
-  public static String stargateHost = "127.0.0.11";
   public boolean enableAuth;
+
   protected final ClusterConnectionInfo backend;
 
+  public static List<Starter> stargateStarters = new ArrayList<>();
+  private static List<String> stargateHosts = new ArrayList<>();
+  public static final Integer numberOfStargateNodes = 3;
```

Comment:
Making it parameterizable will require refactoring the whole testing code to make the `Stargate` runners not shared between test classes. Such a rework will be a substantial effort and will be done in a separate ticket (we have the ticket for it already)
The three, as a default, seems like a reasonable choice for now. It will allow us to validate more scenarios. I followed the same pattern as for CCM now, which is sharing the cluster between tests.